llvm/lib/SYCL/PrepareSYCLOpt.cpp

```diff
@@ -53,6 +58,14 @@ struct PrepareSYCLOpt : public ModulePass {
     }
   }
 
+  void markKernelandNoInlineForFlattening(Module &M) {
```

Comment:
```suggestion
  void markKernelAndNoInlineForFlattening(Module &M) {
```
while the names are rather clear, you can always add some comments in front of each function nevertheless to explain some design,